packages/office-ui-fabric-react/package.json

```diff
@@ -22,6 +22,7 @@
     "@types/enzyme": "^2.7.5",
```

Comment:
Please use `rush change` to add a change file to patch fabric-react and add a nice comment describing this, which will show up in the changelog.